weather.gismeteo/resources/libs/gismeteo.py

```diff
@@ -0,0 +1,250 @@
+# -*- coding: utf-8 -*-
+# License: GPL v.3 https://www.gnu.org/copyleft/gpl.html
+
+from __future__ import unicode_literals
+
+import time
+import calendar
+import requests
+
+import xml.etree.cElementTree as etree
+try:
+    etree.fromstring('<?xml version="1.0"?><foo><bar/></foo>')
+except TypeError:
+    import xml.etree.ElementTree as etree
+
+__all__ = ['GismeteoError', 'GismeteoClient']
+
+
+class GismeteoError(Exception):
+
+    pass
+
+
+class GismeteoClient(object):
+
+    _base_url = 'https://services.gismeteo.ru/inform-service/inf_chrome'
+
+    def __init__(self, lang='en'):
+
+        self._lang = lang
+
+        self._client = requests.Session()
+        self._client.verify = False
```

Comment:
This is not the way to solve SSL issues. You are basically disabling the security of the SSL mechanism.

Reply:
There was this error on some platforms :

> hostname 'services.gismeteo.ru' doesn't match either of '*.gismeteo.net', 'gismeteo.net'

Looks like changing URL from services.gismeteo.ru to services.gismeteo.net solves it. I didn’t think about such decision at that moment